### PR TITLE
Fire SsaCLIControllerNotOK only if status is not OK and not NOT CONFIGURED

### PR DESCRIPTION
**Pull request by Deezzir**
Fixes: #354

### CODE REVIEW

**Review by aieri — APPROVED** [review]

**Review by jneo8 — APPROVED** [review]

File: src/prometheus_alert_rules/ssacli.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ groups:
           LABELS = {{ $labels }}
 
   - alert: SsaCLIControllerNotOK
-    expr: ssacli_controller_info{status != "OK"} == 1
+    expr: ssacli_controller_info{status != "OK", status!="NOT CONFIGURED"} == 1
```
Comment (gabrielcocenza) [inline src/prometheus_alert_rules/ssacli.yaml:30]: This is not a valid Promql expression. 

I will revert this PR.

Please, also add unit tests using the protool

Comment (Deezzir) [reply src/prometheus_alert_rules/ssacli.yaml:30]: It is a valid expression, I tested it

Comment (gabrielcocenza) [reply src/prometheus_alert_rules/ssacli.yaml:30]: How did you test this?

image

It won't break the query, but it always will return empty.
As far as I know you can't use two times `status` and expect this kind of behavior

Comment (gabrielcocenza) [reply src/prometheus_alert_rules/ssacli.yaml:30]: See #376

Comment (Deezzir) [reply src/prometheus_alert_rules/ssacli.yaml:30]: I tested with the same unit test you provided, I just didn't add it to the PR

Comment (gabrielcocenza) [reply src/prometheus_alert_rules/ssacli.yaml:30]: Incredibly it pass the unit tests if I set the `NOT CONFIGURED`. However this is very likely a promtool bug that we should file.

As showed above this kind of query will always return empty in Prometheus so it's useless in a real scenario

**Pull request merged**